src/services/web3.js

```diff
@@ -441,7 +441,7 @@ function checkJNTEvent(data: Object): boolean {
   )
 }
 
-function prepareJNTEvents(data: Array<Object>, assetAddress: AssetAddress): Transactions {
+function prepareJNTEvents(data: Array<Object>): Transactions {
```

Comment:
but you forgot to remove passing of assetAddress from call(s) of prepareJNTEvents